daml-lf/transaction-test-lib/src/main/scala/lf/value/test/ValueGenerators.scala

```diff
@@ -353,6 +354,17 @@ object ValueGenerators {
     )
   }
 
+  /** Makes rollback node with some random child IDs. */
```

Comment:
@stephencompall-DA Could you take a look at this file ?

Reply:
Most tests shouldn't care that the generated transactions are well-formed in any way. If expecting the IDs in the rollback is an exception, then it's worth worrying about; otherwise random IDs should be fine.

Reply:
However, all existing uses of this kind of node-ID generation use "danglingRef" or "malformed" in their name to indicate this, and this practice has not been carried through here.